node_modules/col-assets/lib/api.js

```diff
@@ -64,6 +67,13 @@ var getAssetProfile = module.exports.getAssetProfile = function(ctx, id, increme
 
     asset = asset.toJSON();
     asset.whiteboard_elements = _.pluck(asset.whiteboard_elements, ['element']);
+    asset.exported_whiteboards = _.pluck(asset.exported_whiteboards, ['id', 'title']);
```

Comment:
Do we need to do the `pluck` here as we're already doing `'attributes': ['title', 'id'],` below ?